include/oneapi/tbb/scalable_allocator.h

```diff
@@ -23,6 +23,7 @@
 #include <cstdlib>
 #include <utility>
```

Comment:
It is not specific to the patch but inclusion of scalable_allocator.h into proxy.cpp causes the issue below with gcc 4.8.5. Include `<new>` into scalable_allocator.h (if possible move inclusion of `scalable_allocator.h` up in `test_scalable_allocator.cpp`)
```
src/tbbmalloc_proxy/../../include/oneapi/tbb/scalable_allocator.h:225:29: error: 'bad_alloc' is not a member of 'std'
              throw_exception(std::bad_alloc());
```

Reply:
Is it from the C test? I'm wondering how it would work in that case, since there's no exception in C.

Reply:
It is from tbbmalloc_proxy build with gcc 4.8.5. While tbbmalloc and tbbmalloc_proxy do not depend on C++ runtime they are built with C++ compiler.